data/src/test/java/org/apache/iceberg/data/TestMetricsRowGroupFilter.java

```diff
@@ -211,6 +218,9 @@ public void createOrcInputFile() throws IOException {
         structNotNull.setField("_int_field", INT_MIN_VALUE + i);
         record.setField("_struct_not_null", structNotNull); // struct with int
 
+        record.setField(
+            "_uuid_col", (i % 3 == 0) ? UUID_WITH_ZEROS : (i % 3 == 1) ? UUID_WITH_ONES : null);
```

Comment:
minor: what's the reason for doing the modulo here? why not just write UUID_WITH_ZEROS?

Reply:
The idea was to use different values to make sure the filtering works, but now I think just with_zeros and null are enough. WDYT?